<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatosEmpresa</class>
 <widget class="QDialog" name="DatosEmpresa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,10">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Empresa      </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtEmpresa"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,5,1,2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Domicilio    </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtDomicilio"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>NÂº Patronal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtNumeroPatronal"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,8,1,4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>C. Trabajo   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtCentroTrabajo">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>C.I.F.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtCif"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,8,1,4">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Actividad    </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtActividad"/>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Convenio</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtConvenio"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Volver</string>
   </property>
   <property name="icon">
    <iconset resource="imagenes/imagenes.qrc">
     <normaloff>:/atras.png</normaloff>:/atras.png</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGuardar">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>220</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="icon">
    <iconset resource="imagenes/imagenes.qrc">
     <normaloff>:/save.jpeg</normaloff>:/save.jpeg</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="imagenes/imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
